scripts/contracts.ts

```diff
@@ -102,6 +104,7 @@ export type Contracts = {
     consumerRegistry: ConsumerRegistry;
     priceOracle: PriceOracle;
     polygonDestination: PolygonDestination;
+    rootChainMananger: RootChainManager;
```

Comment:
Since we don't need to deploy root chain manager, we can leave it out from here.

Reply:
Do we still have scenarios to consume it through contract SDK?

Reply:
> Do we still have scenarios to consume it through contract SDK?

I have used `@maticnetwork/maticjs` so we don't need to directly invoke root chain manager